doc/compodoc_sources/guidelines/documentation.md

```diff
@@ -1,9 +1,9 @@
-#Documentation for Developer- how to contribute
+# Documentation for Developers - how to contribute
 * with every push to the master, travis generates this Documentation of the Project via Compodoc automatically. **All you have to do is to Comment your code with basic JSDoc!** Further information: https://compodoc.app/guides/comments.html
 * New code should always be documented while writing!
-* If you want to **add or edit separate additional documentation** like this page, work on branch compodoc and edit the markdown files under doc/compodoc_sources. Make sure to kee the summary.json up-to-date.
+* If you want to **add or edit separate additional documentation** like this page, work on branch compodoc and edit the markdown files under doc/compodoc_sources. If you add a new markdown file you also have to add it to summary.json.
 
-###additional information
+### additional information
 * the script "compodoc" in the package.json is used by travis to generate the documentation!
 * Install compodoc manually with npm i -save -dev @compodoc/compodoc
```

Comment:
for console input please use markdowns `>` highliting:
- Install compodoc manually with
> npm i -save - dev \@compodoc/compodoc

This makes it more clear where the command begins/ends and gives the user the ability to just 'scan' the document for the command he is looking for.